<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_popup_PurchaseList_Itm_Pre</class>
 <widget class="QMainWindow" name="frm_popup_PurchaseList_Itm_Pre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>采购项内容</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_popup_PurchaseList_Itm_ProdName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_popup_PurchaseList_Itm_ProdName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_popup_PurchaseList_Itm_ProdNum">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_popup_PurchaseList_Itm_ProdUnit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_popup_PurchaseList_Itm_ProdNum">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>数量</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_popup_PurchaseList_Itm_ProdUnit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>单位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_popup_PurchaseList_Itm_Pre_Cancel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_popup_PurchaseList_Itm_Pre_Save">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_popup_PurchaseList_Itm_Pre_pick">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>☺</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
